cmake/configure/configure_vectorization.cmake

```diff
@@ -21,19 +21,19 @@
 SET(DEAL_II_EXPAND_REAL_SCALARS_VECTORIZED "VectorizedArray<double,1>" "VectorizedArray<float,1>")
 SET(DEAL_II_EXPAND_FLOAT_VECTORIZED "VectorizedArray<float,1>")
 
-IF(${DEAL_II_COMPILER_VECTORIZATION_LEVEL} GREATER 0)
+IF(${DEAL_II_VECTORIZATION_WIDTH_IN_BITS} GREATER 0)
    SET(DEAL_II_EXPAND_REAL_SCALARS_VECTORIZED 
       "${DEAL_II_EXPAND_REAL_SCALARS_VECTORIZED}" "VectorizedArray<double,2>" "VectorizedArray<float,4>")
    SET(DEAL_II_EXPAND_FLOAT_VECTORIZED  "${DEAL_II_EXPAND_FLOAT_VECTORIZED}" "VectorizedArray<float,4>")
 ENDIF()
 
-IF((${DEAL_II_COMPILER_VECTORIZATION_LEVEL} GREATER 1) AND ( DEAL_II_HAVE_AVX OR DEAL_II_HAVE_ALTIVEC))
```

Comment:
what happened with altivec here?

Reply:
AltiVec is 128 bits, so the condition cannot be true, and the altivec case goes to the `VECTORIZATION_WIDTH_IN_BITS > 0` case. It took me a while to understand the condition, but I think we simply missed it; in any case, I think we could also skip the `AVX` and `AVX512` parts here because the macro tells us what we have.